xclim/indices/fwi.py

```diff
@@ -607,6 +649,64 @@ def _fire_weather_calc(*args, **params):
     return tuple(ind_data.values())
 
 
+def _fwi_arg_compressor(
```

Comment:
This compressor / decompressor approach does not feel pythonic. I understand this is what allows users to request different combination of indices, correct ? 

What are our other options ?

Reply:
Yes, the goal was to only pass the arrays that are needed. `xr.apply_ufunc` can only accept arrays that are defined at least over the `input_core_dims` dimensions. Another option is to set unnecessary arrays to empty arrays and empty tuples in `input_core_dims`. 
It might need less complexity in functions. I'll look into that.

Reply:
Turns out I was wrong. Passing "None" to `apply_ufunc` is not a problem. I removed the `compressor` and `decompressor` functions and added simpler verification in `fire_weather_ufunc`.